File: judge-runtime/spec/lang.dart/validator.dart
class Validator {
  String swapFirstAndLastWords(String s) {
    List<String> words = s.split(' ');

    if (words.length <= 1) {
      return s;
    }

    String temp = words[0];
    words[0] = words[words.length - 1];
    words[words.length - 1] = temp;

    return words.join(' ');
  }
}

File: judge-runtime/spec/lang.ts/validator.ts
export class Validator {
    public swapFirstAndLastWords(s: string): string {
        const words = s.split(' ');

        if (words.length <= 1) {
            return s;
        }

        [words[0], words[words.length - 1]] = [words[words.length - 1], words[0]];

        return words.join(' ');
    }
}
